### PR TITLE
OPS-967: Import ideas from official memgraph chart

### PR DESCRIPTION
**Pull request by enys**

### CODE REVIEW

**Comment by thibveni** [comment]
@enys I don't remember, why are we not using official memgraph chart and adding subchart on it to add our specific needs ?

**Review by thibveni — APPROVED** [review]

File: charts/memgraph/templates/statefulset.yaml [hunk 1]
```diff
@@ -40,6 +40,11 @@ spec:
           image: "{{ $imageName }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command: ["/usr/lib/memgraph/memgraph"]
+          args:
+          {{- range .Values.memgraphConfig }}
+          - {{ . | quote }}
+          {{- end }}
+
```
Comment (thibveni) [inline charts/memgraph/templates/statefulset.yaml:47]: i suppose you already tested it with template command

Comment (enys) [reply charts/memgraph/templates/statefulset.yaml:47]: I've been actively using Tilt for the import subjects, yes.

**Comment by enys** [comment]
> @enys I don't remember, why are we not using official memgraph chart and adding subchart on it to add our specific needs ?

It didn't exist, and I actually shared this with memgraph at the time.
The official one, is still a bit limited today.

**Pull request merged**